packages/block-library/src/breadcrumbs/index.php

```diff
@@ -169,14 +176,15 @@ function block_core_breadcrumbs_is_paged() {
  * Creates a "Page X" breadcrumb item for paginated views.
  *
  * @since 6.9.0
- *
+ * @param string $query_var Optional. Query variable to get current page number. Default 'paged'.
  * @return string The "Page X" breadcrumb HTML.
  */
-function block_core_breadcrumbs_create_page_number_item() {
-	$paged = (int) get_query_var( 'paged' );
+function block_core_breadcrumbs_create_page_number_item( $query_var = 'paged' ) {
+	$paged = (int) get_query_var( $query_var );
+
 	return block_core_breadcrumbs_create_current_item(
 		/* translators: %s: page number */
-		sprintf( __( 'Page %s' ), number_format_i18n( $paged ) )
+		sprintf( __( 'Page %s', 'gutenberg' ), number_format_i18n( $paged ) )
```

Comment:
```suggestion
		sprintf( __( 'Page %s' ), number_format_i18n( $paged ) )
```

For core blocks we don't need the GB context, as they get the `default`.